cpp/src/column/column_device_view.cu

```diff
@@ -41,7 +41,7 @@ void column_device_view::destroy() {
 column_device_view::column_device_view( column_view source, void * h_ptr, void* d_ptr )
     : detail::column_device_view_base{source.type(),       source.size(),
                                       source.head(),       source.null_mask(),
-                                      source.null_count(), source.offset()},
+                                      source.offset()},
```

Comment:
@nvdbaranec I think you incorrectly resolved conflicts in this file when you merged branch-0.14. It seems @davidwendt's PR #4783 moved the contents of this constructor into a helper function and moved this constructor later in the file. I just tried to resolve conflicts on this PR (4799) and found it very confusing because of those changes.  Please go back and fix up the merge.

Reply:
This wasn't a bad local merge.  That PR (4783) just got commited at an unfortunate time (after the last update to this got pushed).  Updated.